internal/test/issues/issue-1039/issue_test.go

```diff
@@ -0,0 +1,298 @@
+package issue1039
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	issue1039nullable "github.com/deepmap/oapi-codegen/v2/internal/test/issues/issue-1039/nullable-disabled"
+	"github.com/deepmap/oapi-codegen/v2/pkg/codegen"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+//go:embed spec.yaml
+var spec []byte
+
+func ToStringPointer(str string) *string {
+	return &str
+}
+
+func TestIssue(t *testing.T) {
+	swagger, err := openapi3.NewLoader().LoadFromData(spec)
+	require.NoError(t, err)
+
+	opts := codegen.Configuration{
+		PackageName: "issue1039",
+		Generate: codegen.GenerateOptions{
+			ChiServer:    true,
+			Client:       true,
+			Models:       true,
+			EmbeddedSpec: true,
+			NullableType: true,
+		},
+	}
+
+	_, err = codegen.Generate(swagger, opts)
+	require.NoError(t, err)
+}
```

Comment:
Was this just to check that generating the code works? If so, this shouldn't be needed - by having the `go:generate`s in this package, that's good enough, unless there's something else you're trying to test?

Reply:
Yes. Removed it.